packages/react-native-bridge/android/build.gradle

```diff
@@ -137,7 +143,24 @@ repositories {
 }
 
 dependencies {
-    implementation project(':@wordpress_react-native-aztec')
+    // Embedding Aztec seems to break the build from source when running from
+    // the WordPress Android project. That shouldn't be the case. While we
+    // investigate, embed only when making a binary build.
+    //
+    // Note that this also means that we cannot run an `assemble` task for
+    // WordPress Android while building from source, because that would result
```

Comment:
> we cannot run an `assemble` task for WordPress Android while building from source

This is a nasty little gotcha. I'm guessing this happens only when the unqualified `assemble` task is run (as compared to something like `assembleWasabiDebug` which will not try to directly build the bridge since the bridge project doesn't have a task with that name) because it tries to assemble this dependency. FWIW, just noting that a quick way to get this error is to run `./gradlew :@wordpress_react-native-bridge:assemble` from WPAndroid.

It is interesting how the full error message points to hermes being the problem:

> Direct local .aar file dependencies are not supported when building an AAR. The resulting AAR would be broken because the classes and Android resources from any local .aar file dependencies would not be packaged in the resulting AAR. Previous versions of the Android Gradle Plugin produce broken AARs in this case too (despite not throwing this error). The following direct local .aar file dependencies of the :@wordpress_react-native-bridge project caused this error: /Users/matt/dev/a8c/WordPress-Android/libs/gutenberg-mobile/gutenberg/node_modules/hermes-engine/android/hermes-debug.aar

I wonder if this actually might be calling out the issue that was causing the app to crash because of the missing hermes dependency (and that we "fixed" by manually including the hermes aars in the WPAndroid repo). 🤔 Maybe fixing the hermes issue would also fix this issue (and then we wouldn't have to include the hermes dependencies in the WPAndroid--and keep them up-to-date).

Hopefully, we can figure this out, but if we can't, do you think it would be worthwhile to have the WPAndroid build perform a check and abort early (with a descriptive error message) if someone tries to do a build from source that would assemble this dependency?

Reply:
>  do you think it would be worthwhile to have the WPAndroid build perform a check and abort early (with a descriptive error message) if someone tries to do a build from source that would assemble this dependency?

For sure 👍